packages/core/src/payment/strategies/amazon-pay/amazon-pay-payment-strategy.spec.ts

```diff
@@ -349,7 +349,7 @@ describe('AmazonPayPaymentStrategy', () => {
         }
     });
 
-    it('returns error response if order submission fails', async () => {
+    it.skip('returns error response if order submission fails', async () => {
```

Comment:
```suggestion
    it('returns error response if order submission fails', async () => {
```

You forgot to remove `skip`?

Reply:
This is intentional, I have ping @ggarcia-bc to see if this test has issues with typescript upgrade 😢 Since I was not able to debug it straightforward as there is a callback to amazon method that doesn't seem to be firing.

Reply:
https://github.com/bigcommerce/checkout-sdk-js/pull/1692#issuecomment-1317508847